ani-cli

```diff
@@ -224,7 +224,9 @@ update_history() {
 download() {
     case $1 in
         *m3u8*)
-            if command -v "yt-dlp" >/dev/null; then
+            if command -v "N_m3u8DL-RE" >/dev/null; then
+                N_m3u8DL-RE --log-level ERROR --thread-count 10 --download-retry-count 10 "$1" --save-dir "$download_dir" --save-name "$2"
```

Comment:
Is there any particular reason to run it with less threads than yt-dlp? (10 as oppose to 16)
Was there a process in selecting these two numbers or are they just rules of thumb?

Reply:
this is only temporary until approved add N_m3u8DL-RE